imagetool/src/main/resources/docker-files/install-middleware.mustache

```diff
@@ -35,9 +35,13 @@ USER {{userid}}
 RUN echo "INSTALLING MIDDLEWARE" \
 {{#installPackages}}
     && echo "INSTALLING {{type}}" \
-    && {{#isZip}}unzip -q {{{tempDir}}}/{{installerFilename}} "*.jar" -d {{{tempDir}}} &&{{/isZip}} \
+    && {{^isBin}}{{#isZip}}unzip -q {{{tempDir}}}/{{installerFilename}} "*.jar" -d {{{tempDir}}} &&{{/isZip}} \
```

Comment:
Since the unzip applies to both isBin and not-isBin, why not pull that out of the two groups and put it first?

Reply:
Moved unzip of both jar&bin installer in to a single block at the top.
https://github.com/brahankv/weblogic-image-tool/commit/213ca302d27ccbcad696ac31ed635148a6d7921b